components/combobox/__tests__/combobox.browser-test.jsx

```diff
@@ -211,12 +211,14 @@ describe('SLDSCombobox', function () {
 			destroyMountNode({ wrapper, mountNode });
 		});
 
-		it('has aria-haspopup, aria-expanded is false when closed, aria-expanded is true when open, ', function () {
+		it('has aria-haspopup, aria-expanded is false when closed, aria-expanded is true when open, has aria-describedby', function () {
 			wrapper = mount(<DemoComponent multiple />, { attachTo: mountNode });
 			const nodes = getNodes({ wrapper });
 			expect(nodes.combobox.node.getAttribute('aria-haspopup')).to.equal(
 				'listbox'
 			);
+			expect(nodes.combobox.node.getAttribute('aria-describedby')).to.not.be
```

Comment:
This can be removed. `aria-describedby` is a static relationship that can be tested with Jest markup tests.